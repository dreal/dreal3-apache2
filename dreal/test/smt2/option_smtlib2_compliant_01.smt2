(set-logic QF_NRA)
(set-option :smtlib2_compliant true)
(set-option :produce-models true)
(declare-const z Real)
(define-fun minus ((x Real) (y Real)) Real (- x y))
(assert (= (minus 4 3) z))
(check-sat)
(get-option :precision)
(get-value (z))
(exit)
