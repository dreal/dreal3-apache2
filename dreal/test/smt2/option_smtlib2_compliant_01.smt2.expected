delta-sat
0.001
(
	(z 1)
)
